docs/_integration-manager/camel-routes/camel-history-management.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: default
+title: Camel File Processing History Management
+nav_order: 5
+parent: File Folder Listener
```

Comment:
parent should be Camel Route Configuration, not File Folder Listener

Reply:
Done